crates/cargo-test-support/src/compare.rs

```diff
@@ -178,15 +165,35 @@ pub fn assert_e2e() -> snapbox::Assert {
         regex::Regex::new("home/\\.cargo/registry/src/-(?<redacted>[a-z0-9]+)").unwrap(),
     )
     .unwrap();
-    snapbox::Assert::new()
-        .action_env(snapbox::assert::DEFAULT_ACTION_ENV)
-        .redact_with(subs)
+    subs.insert(
+        "[HASH]",
+        regex::Regex::new(r"[a-z0-9\-_]+-(?<redacted>[0-9a-f]{16})").unwrap(),
```

Comment:
This is still fairly broad. Is there something else about these situations we can pick up on, like use of `/`?

Reply:
I switched to check for a `/` in front